lib/timers.js

```diff
@@ -317,6 +320,11 @@ exports.setTimeout = function(callback, after) {
   after *= 1; // coalesce to number or NaN
 
   if (!(after >= 1 && after <= TIMEOUT_MAX)) {
+    if (after > TIMEOUT_MAX) {
```

Comment:
Lots of code out there that does either `setTimeout(cb)` or `setTimeout(cb, 0)` -- even our own tests.